x-pack/packages/security-solution/navigation/README.mdx

```diff
@@ -0,0 +1,4 @@
+## Security Solution Navigation
+
+This package provides resources to be used for Security Solution navigation
```

Comment:
That will be useful to extend Readme with examples how to use this package, but we can do this as follow up.